adam/curriculum/preposition_curriculum.py

```diff
@@ -139,19 +147,20 @@ def _over_template(
     *,
     is_training: bool,
     is_distal: bool,
-    syntax_hints: Iterable[str] = [],
+    syntax_hints: Iterable[str] = immutableset(),
+    background_relations: Iterable[TemplateObjectVariable] = immutableset(),
 ) -> Phase1SituationTemplate:
     handle = "training" if is_training else "testing"
-    # TODO: currently this hack keeps old implementation for English that hasn't solved https://github.com/isi-vista/adam/issues/802
-    # and returns new implementation for Chinese that does solve this
+    relations = [
```

Comment:
Commented problem resolved?

Reply:
Yes, at least according to the git issue.

Reply:
Right. What I was confused about was whether this reference meant that "this is the problem that needs to be fixed" (and since the issue is, the problem is) or "this problem was mostly fixed, which is why the issue is resolved, but this specific case was not fixed, and here is the issue which gives more context about why/the way in which it was not fixed." Though if it were the latter case then there should have been a separate issue, so this is probably resolved.